webpack/JobInvocationDetail/JobAdditionInfo.js

```diff
@@ -0,0 +1,399 @@
+                  </DataListCell>,
+                  <DataListCell width={4} key={1}>
+                    {recurrence.iteration}
+                  </DataListCell>,
+                ]}
+              />
+            </DataListItemRow>
+          </DataListItem>
+          <DataListItem>
+            <DataListItemRow>
+              <DataListItemCells
+                dataListCells={[
+                  <DataListCell width={1} key={0}>
+                    {__('Iteration limit')}
+                  </DataListCell>,
+                  <DataListCell width={4} key={1}>
+                    {recurrence.max_iteration}
+                  </DataListCell>,
+                ]}
+              />
+            </DataListItemRow>
+          </DataListItem>
+          <DataListItem>
+            <DataListItemRow>
+              <DataListItemCells
+                dataListCells={[
+                  <DataListCell width={1} key={0}>
+                    {__('Repeat until')}
+                  </DataListCell>,
+                  <DataListCell width={4} key={1}>
+                    {recurrence.end_time}
+                  </DataListCell>,
+                ]}
+              />
+            </DataListItemRow>
+          </DataListItem>
+          {/* TODO <DataListCell width={1} key={0}>{__('State')}</DataListCell>
+            <DataListCell width={1} key={0}>
+              {recurring_logic_state(recurring_logic)}
+            </DataListCell> */}
+          <DataListItem>
+            <DataListItemRow>
+              <DataListItemCells
+                dataListCells={[
+                  <DataListCell width={1} key={0}>
+                    {__('Purpose')}
+                  </DataListCell>,
+                  <DataListCell width={4} key={1}>
+                    {recurrence.purpose}
+                  </DataListCell>,
+                ]}
+              />
+            </DataListItemRow>
+          </DataListItem>
+          <DataListItem>
+            <DataListItemRow>
+              <DataListItemCells
+                dataListCells={[
+                  <DataListCell width={1} key={0}>
+                    {__('Task count')}
+                  </DataListCell>,
+                  <DataListCell width={4} key={1}>
+                    {recurrence.task_count}
+                  </DataListCell>,
+                ]}
+              />
+            </DataListItemRow>
+          </DataListItem>
+        </>
+      </DataList>
+    </ExpandableSection>
+  );
+};
+const TargetHosts = ({ data }) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const {
+    targeting,
+    job_location: location,
+    job_organization: organization,
+  } = data;
+
+  // manual selection using static query
+  // execution order
+  //
+  // TODO is needed? bookmark_id is always null in the wizard, also why is knowing the bookmark relevent?
+  // const targetingSelectioning = targeting.bookmark_id ? `{__('Bookmark') ${targeting.bookmark_name}` :  __('Manual selection');
```

Comment:
@adamruzicka _card_target_hosts does:
``` rb
    <% if job_invocation.bookmark.present? %>
      <%= _('Bookmark') %> <%= job_invocation.bookmark.name %><br>
    <% else %>
      <%= _('Manual selection') %>
    <% end %>
    <%= _('using ') %>
    <strong><%= _(Targeting::TYPES[job_invocation.targeting.targeting_type]).downcase %></strong>
    <pre><%= job_invocation.targeting.search_query %></pre>
    </p>
    ```
    but in the job wizard we always send null in bookmarks, but users can send bookmarks in api post requests.
    Should we show bookmark info here, skip it, or work later to add it?

Reply:
I'd vote for showing it and (eventually) make the wizard send the information as well

Reply:
Showing bookmark name now